packages/core/useConfirmDialog/index.ts

```diff
@@ -0,0 +1,67 @@
+import { Ref } from 'vue-demi'
+import { createEventHook, EventHook, Fn } from '@vueuse/core'
+
+export interface useConfirmDialogReturn<D, E> {
```

Comment:
We could also use default types like `<D = any, E = any>` or maybe `<D = unknown, E = unknown>`

Reply:
Oh nevermind... have now read line 43

Reply:
I have already used this on the function itself